addons/web/static/src/scss/list_view.scss

```diff
@@ -85,7 +84,11 @@
             }
         }
 
-        .o_list_record_selector, .o_list_record_remove, .o_handle_cell {
+        .o_list_record_selector {
+            width: 41px;
+            padding-left: $o-horizontal-padding;
+        }
+        .o_list_record_remove, .o_handle_cell {
```

Comment:
Are you sure that those changes in fields.scss and list_view.scss have no negative impact in enterprise?

Reply:
There is no test to prove it but I haven't spotted any difference in the list I checked (list view readonly & editable and x2many lists)